packages/ffe-cards-react/src/components/Subtext.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import classNames from 'classnames';
+import { string } from 'prop-types';
+
+import ComponentBase from './ComponentBase';
+
+const Subtext = ({ className, ...rest }) => (
```

Comment:
Is `<SubTitle />` a better name? Subtext is, according to [Wikipedia](https://en.wikipedia.org/wiki/Subtext):

> Subtext or undertone is any content of a creative work which is not announced explicitly by the characters or author, but is implicit or becomes something understood by the observer of the work as the production unfolds.

Reply:
Good question. I took all the sub component names from @adidick's https://github.com/SpareBank1/designsystem/pull/262#issuecomment-392735553

Reply:
These names are totally up for improvement so I'm open to any suggestions that make more sense from your perspective.

Reply:
To help reduce misunderstandings, I think it's an advantage if ux and developers use the same component names as much as possible. Whether it be [Subtitle](https://en.wikipedia.org/wiki/Subtitle_(titling)), Subtext or something else.

Any other opinions on which name we should favor?

Reply:
We call them "lead paragraphs" in the regular typography but I suppose that kind of depends on how much text we're going to be putting into them.